ui/js/admin.js

```diff
@@ -1,39 +1,37 @@
 const signup = document.getElementById("signup");
 const signupModal = document.getElementsByClassName("signup-modal")[0];
-const signupBox = document.getElementsByClassName("signup-box")[0];
 const login = document.getElementById("signin");
 const loginModal = document.getElementsByClassName("login-modal")[0];
-const loginBox = document.getElementsByClassName("login-box")[0];
 const fastfoodfast = document.getElementsByClassName("fastfoodfast")[0];
 
 //Open modal for sign up
 signup.addEventListener("click", () => {
-    signupModal.style.display = "block";
+	signupModal.style.display = "block";
 });
 
 // close modal onclick outside for signup
 function closeSignupModal(m) {
```

Comment:
#### Critical
This is a function, not a class method. Use ES6 syntax. You were required to use ES6. Redeclare using arrow function.